lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1182,7 +1182,7 @@ def writeHatches(self):
                             0, 0, sidelen, sidelen, Op.rectangle,
                             Op.fill)
 
-            self.output(0.1, Op.setlinewidth)
+            self.output(rcParams['hatch.linewidth'], Op.setlinewidth)
```

Comment:
How significant of a performance hit is it to be constantly re-querying this dictionary?

Also, is it the right thing to access the rcParams this late in the draw process? We are pretty bad at being consistent, but I don't recall doing this sort of rcParam access this late anywhere else. What if someone uses an rc context to set up the hatches, but the savefig call is outside that context?